src/twisted/topfiles/9126.feature

```diff
@@ -0,0 +1 @@
+twisted.python.url and twisted.python._url were modified to rely on hyperlink, a new package based on the Twisted URL implementation. Hyperlink adds support for IPv6 (fixing #8069), correct username/password encoding, better scheme/netloc inference, improved URL.click() behavior (fixing #8184), and more. For full docs see hyperlink.readthedocs.io and the CHANGELOG in the hyperlink GitHub repo.
```

Comment:
You should break each bug fixed into its own topfile.   Whoever merges this should put all the tickets into the `Fixes` line, as @rodrigc [did](https://github.com/twisted/twisted/commit/e116bbe34dc3a3855a674043bc5f30d7e6417c7f) in #630.  Something like:

```
Fixes: ticket:8069 , ticket:8184 , ticket:9126
```

It'd be great if we could figure out _all_ the tickets this closes, but we can start with the ones you've identified.